proceedings-of-the-royal-society-b.csl

```diff
@@ -49,10 +49,8 @@
   </macro>
   <macro name="publisher">
```

Comment:
given how popular this style is, we should also do the publisher/publisher place thing right, i.e. with a group with delimiter=": "